biogrid/apps/api/src/app/services/Biogrid.ts

```diff
@@ -71,8 +71,12 @@ export async function simulateNewBiogrid(
   const biobrain = BioBrain.Instance;
   const initState = biogrid.getSystemState();
   const statesJson = [biogrid.getJsonGraphDetails()];
+  const currentDate = body.startDate;
   for (let i = 0; i < constants.simulation.NUMBER_OF_SIM_CYCLES; i++) {
```

Comment:
can we clarify the naming of NUMBER_OF_SIM_CYCLES and change it to NUMBER_OF_SIM_HOURS instead?